src/main/java/org/bytedeco/javacpp/tools/Context.java

```diff
@@ -62,6 +63,7 @@
     boolean inaccessible = false;
     boolean objectify = false;
     boolean virtualize = false;
+    boolean readOnlyMembers = false;
```

Comment:
For consistency, what about naming this "constify"? (It's actually a term that C++ programmers use!)

Reply:
Or maybe just "immutable"?

Reply:
But that way this property would also have to disable non-const C++ methods (and maybe some other things too) otherwise it would be confusing.

Although, current name also doesn't make it 100% clear that it deals only with data members, since methods in C++ are technically called member functions.

Reply:
Right w.r.t to "const", but "immutable" can only refer to *data* and usually only to "properties", even in the case of C++. Functions, or even variables for that matter, are not "properties", because well we can't do things like make them immutable. :)

Reply:
I see, we can't currently apply this to individual fields, but ideally it's something that it should let us do. I guess we can wait until someone asks for it before implementing this though.